backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepositoryFactory.java

```diff
@@ -488,19 +500,19 @@ private SyncToRemoteStatus deriveSyncToRemoteStatus(IGitCoreLocalBranch coreLoca
         String newIdHash = e.getNewCommitHash().getHashString();
 
         if (e.getNewCommitHash().equals(entryToExcludeNewId)) {
-          LOG.trace(() -> "Exclude ${newIdHash} because it has the same hash as first entry");
+          LOG.trace(() -> "Exclude ${e} because it has the same hash as first entry");
```

Comment:
What is the difference?

Reply:
`e` is the entire entry, so not just new id but also old id and subject